packages/next/src/next.ts

```diff
@@ -0,0 +1,70 @@
+// Must be in an input folder for mkdist to build
+import { createElement, useEffect, useReducer } from 'react';
+import type { ComponentProps, FunctionComponent } from 'react';
+
+let millionModule: any = null;
```

Comment:
```suggestion
let millionModule: typeof import('million/react') | null = null;
```

Does this type help?

Reply:
wow i had no idea this was possible, thank you